_posts/en/newsletters/2019-07-17-newsletter.md

```diff
@@ -0,0 +1,98 @@
+gossip protocol. Also included are our regular sections on bech32
+sending support and notable changes in popular Bitcoin infrastructure
+projects.
+
+{% comment %}<!-- include references.md below the fold but above any Jekyll/Liquid variables-->{% endcomment %}
+{% include references.md %}
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Gossip update proposal:** Rusty Russell made a [short proposal][v2
+  gossip proposal] to update the gossip protocol that LN nodes use to
+  announce what channels they have available for routing payments and
+  what capabilities those channels currently support.  Most notably, the
+  proposal significantly reduces the byte size of messages through two
+  mechanisms: schnorr signatures and optional message extensions.
+  Schnorr signatures based on [bip-schnorr][] save a few bytes because
+  of their more efficient encoding compared to DER-encoded ECDSA
+  signatures, but their most significant savings in the gossip
+  improvement proposal comes from the ability to aggregate the two
+  signatures for a channel announcement into a single signature using
+  [MuSig][].  Optional message extensions using Tag-Length-Value (TLV)
+  records allow omitting unnecessary details when the protocol defaults
+  are being used (for more information about TLV, see the *notable
+  code and documentation changes* section below).
+
+## Bech32 sending support
+
+*Week 18 of 24 in a [series][bech32 series] about allowing the people
+you pay to access all of segwit's benefits.*
+
+{% comment %}<!-- weekly reminder for harding: check Bech32 Adoption
+wiki page for changes -->{% endcomment %}
+
+{% include specials/bech32/18-real-fees.md %}
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[LND][lnd repo], [C-Lightning][c-lightning repo], [Eclair][eclair repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #15277][] allows the Linux versions of Bitcoin Core to
+  be deterministically compiled using [GNU Guix][] (pronounced "geeks").
+  This requires much less setup than the currently-still-supported
+  mechanism based on [Gitian][] and so it is hoped that it will lead to a
+  greater number of users being able to independently verify that the
+  release binaries are derived solely from the Bitcoin Core source code
+  and its dependencies.  Additionally, Guix requires fewer build
+  environment dependencies and there is ongoing work to essentially
+  eliminate its need for any pre-compiled binaries in the typical build
+  toolchain, both of which make the build system much easier to audit.
+  Altogether this should reduce the amount of trust users need to place
+  in the developers of Bitcoin Core and the software used to build
+  Bitcoin Core.  Although this merged PR only builds Linux binaries,
+  it is expected that support for Windows and macOS will
+  follow.
```

Comment:
Perhaps link to Carl's talk at Breaking Bitcoin (https://www.youtube.com/watch?v=DKOG0BQMmmg&feature=youtu.be&t=19828) for people who want to learn more about bootstrappable and deterministic builds.